### PR TITLE
fix: TrezorConnect default import for some build environments

### PR DESCRIPTION
**Pull request by bucko13**

### CODE REVIEW

File: packages/caravan-wallets/src/trezor.ts [hunk 1]
```diff
@@ -111,6 +129,7 @@ const TREZOR_DEV =
   env_variables.VITE_TREZOR_DEV;
 
 try {
+  console.log("TrezorConnect:", TrezorConnect);
```
Comment (sashafklein) [inline packages/caravan-wallets/src/trezor.ts:132]: Remove

File: packages/caravan-wallets/src/trezor.ts [hunk 2]
```diff
@@ -804,6 +823,7 @@ export class TrezorSignMultisigTransaction extends TrezorInteraction {
   }
 }
 
+// eslint-disable-next-line multiline-comment-style
```
Comment (sashafklein) [inline packages/caravan-wallets/src/trezor.ts:826]: Seems these changes (this and the two commented imports) can be removed

**Review by sashafklein — CHANGES_REQUESTED** [review]

**Review by sashafklein — DISMISSED** [review]

File: packages/caravan-wallets/src/trezor.ts (hunk 1 above)
Comment (bucko13) [reply packages/caravan-wallets/src/trezor.ts]: [8156e72](https://github.com/caravan-bitcoin/caravan/pull/41/commits/8156e7202c350205b2d5ed6b379054f26b45c60d)

File: packages/caravan-wallets/src/trezor.ts (hunk 2 above)
Comment (bucko13) [reply packages/caravan-wallets/src/trezor.ts:825]: Unsure why eslint called just this comment out. Didn't really want to dig into it and potentially muck up the diff even more. Maybe something about spacing?

Comment (bucko13) [reply packages/caravan-wallets/src/trezor.ts]: cleaned up a superfluous rule: [bbc5543](https://github.com/caravan-bitcoin/caravan/pull/41/commits/bbc554319ff886c0f1946b1cb5ed9213c2dfe0f5)

**Pull request merged**